templates/data/custom_dataloader_plugin/upload.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+cd $(dirname $0)
+set -e
```

Comment:
Move this to the first line, otherwise a failed `cd` command doesn't abort the script.